docs/config.md

```diff
@@ -1467,23 +1467,33 @@ On all platforms, the user's global `jj` configurations are loaded in the follow
 precedence order (with later configs overriding earlier ones):
 
 - `$HOME/.jjconfig.toml`
-- `$XDG_CONFIG_HOME/jj/config.toml` (preferred)
-- `$XDG_CONFIG_HOME/jj/conf.d/*.toml`
+- `$PLATFORM_SPECIFIC/jj/config.toml` (preferred)
+- `$PLATFORM_SPECIFIC/jj/conf.d/*.toml`
```

Comment:
These are not real env variables so I would avoid writing them as such, consider something like:

```suggestion
- `<PLATFORM_SPECIFIC>/jj/config.toml` (preferred)
- `<PLATFORM_SPECIFIC>/jj/conf.d/*.toml`
```

Reply:
Good idea, thanks!